lib/services/sso_mfa.go

```diff
@@ -34,5 +34,11 @@ type SSOMFASessionData struct {
 	// ConnectorType is SSO type of the corresponding Auth connector (SAML, OIDC).
 	ConnectorType string `json:"connector_type,omitempty"`
 	// ChallengeExtensions are Teleport extensions that apply to this SSO MFA session.
-	ChallengeExtensions *mfav1.ChallengeExtensions `json:"challenge_extensions,omitempty"`
+	ChallengeExtensions *ChallengeExtensions `json:"challenge_extensions"`
+}
+
+// ChallengeExtensions is a json struct for [mfav1.ChallengeExtensions].
+type ChallengeExtensions struct {
+	Scope      mfav1.ChallengeScope      `json:"scope,omitempty"`
+	AllowReuse mfav1.ChallengeAllowReuse `json:"allow_reuse,omitempty"`
```

Comment:
Is `UserVerificationRequirement` deliberately omitted from here?

Reply:
+1, could we document the omission?